.github/workflows/ci.yml

```diff
@@ -737,6 +737,42 @@ jobs:
         run: breeze ci fix-ownership
         if: always()
 
+  test-airflow-release-commands:
+    timeout-minutes: 80
+    name: "Test Airflow release commands"
+    runs-on: "${{needs.build-info.outputs.runs-on}}"
+    needs: [build-info, wait-for-ci-images]
+    env:
+      RUNS_ON: "${{needs.build-info.outputs.runs-on}}"
+      PYTHON_MAJOR_MINOR_VERSION: "${{needs.build-info.outputs.default-python-version}}"
+    steps:
+      - name: Cleanup repo
+        run: docker run -v "${GITHUB_WORKSPACE}:/workspace" -u 0:0 bash -c "rm -rf /workspace/*"
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: >
+          Prepare breeze & CI image: ${{needs.build-info.outputs.default-python-version}}:${{env.IMAGE_TAG}}
+        uses: ./.github/actions/prepare_breeze_and_image
+      - name: "Cleanup dist files"
+        run: rm -fv ./dist/*
+      - name: Install subversion
+        shell: bash
+        run: sudo apt-get install libsvn-dev libapache2-mod-svn subversion-tools subversion
```

Comment:
Hi @potiuk , what's the correct way to install subversion in the CI? I want the step where the ASF repo was cloned to be run in the CI but I can't get the subversion installed

Reply:
I can add svn to our AMI one moment

Reply:
I mean I will have to rebuild it so hold on

Reply:
You can try now and it should be fine. Our self-hosted images should have subversion installed. 

But you should add a step with this condition:

```
    if: needs.build-info.outputs.runs-in == 'self-hosted'
````

Only self-hosted has SVN installed, and I cannot imagine a non-committer changing it, so this should be ok.